iDiff/tests/fileDiff_test_processor.py

```diff
@@ -27,7 +27,7 @@ def _trim_file_names(files):
 
 
 def _trim_layer_hash(filename):
-    hash_match = re.match(r'^([a-z|0-9]{64})/', filename)
+    hash_match = re.match(r'^([a-z|0-9])/', filename)
```

Comment:
We definitely don't need the fileDiff post-processor but until adjustments are made to the node differ (in the works to generalize the multi version diff utils) we still need it for node integration tests.